geometry_msgs/Point context

---

prompTraj prediction